app/models/induction_record.rb

```diff
@@ -65,6 +66,12 @@ def self.latest
   delegate :lead_provider, to: :induction_programme, allow_nil: true
   delegate :lead_provider_name, to: :induction_programme, allow_nil: true
 
+  def active?(lead_provider)
```

Comment:
where is this new method used? I don't see it referenced anywhere or am i missing it?